index.html

```diff
@@ -6,7 +6,37 @@
 <p>18F Guides is the repository for best practices across our teams. 
 Our guides are open source, and you're free to use them as you wish.
 Our hope is that other digital service teams — both inside and outside the government — 
-will adopt or modify the practices outlined here. </p> 
+will adopt or modify the practices outlined here.</p>
+
+<p>Here's a curated list of guides found both here and elswhere that will help you get started:</p>
+
+##Management
```

Comment:
Can you try to fashion these into the sidebar via the `navigation:` section in the `_config.yml` file per: https://pages.18f.gov/guides-template/add-a-new-page/make-a-child-page/

Also interested in what @emileighoutlaw, @kategarklavs, and/or @awfrancisco in particular think of this proposed architecture.